data/json/items/gun/20x66mm.json

```diff
@@ -1,26 +1,23 @@
 [
   {
     "id": "rm120c",
+    "copy-from": "shotgun_base",
     "type": "GUN",
     "name": "RM120c shotgun",
     "description": "Utilizing a powerful and unusual caliber, the Rivtech RM120c caseless pump-action shotgun was designed for hunting big game, with durability and ease of use as priorities.",
     "weight": 1880,
     "volume": 9,
     "price": 310000,
-    "bashing": 8,
-    "to_hit": -1,
     "material": [ "superalloy", "ceramic" ],
-    "symbol": "(",
-    "color": "dark_gray",
-    "skill": "shotgun",
     "ammo": "20x66mm",
     "range": 6,
     "ranged_damage": 4,
     "dispersion": 45,
-    "sight_dispersion": 90,
-    "aim_speed": 4,
     "recoil": 75,
-    "durability": 9,
+    "relative": {
+        "aim_speed": -2,
+	"durability": 1
```

Comment:
We really need a `json-check` to reject tabs in the JSON to cut down on such errors